ci-operator/step-registry/gather/gather-with-network-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
+  as: gather-with-network
```

Comment:
I am not perfect at this either and you might want to think of a slightly different way to name this but I think it would be 

```
chain:
  as: gather-gather-with-network
```
https://steps.svc.ci.openshift.org/help#layout